recipes/dms/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dms" %}
+{% set version = "v1.1" %}
+{% set sha256 = "c368b450cdf1a7f95cb835f7b8190305317f53f26cf34e10244ea817bad77da6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/qdu-zyf/dynamic-meta-storms/archive/master.zip
```

Comment:
Can you tag a release on either your fork or the original repository?

Reply:
Yes, I have added the version tag on my fork repository and updated the "meta.yaml" file. Thanks so much!